.github/workflows/production.yml

```
@@ -0,0 +1,46 @@
+name: Deploy Production Branch
+
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron:  '0 6,18 * * *'
+
+jobs:
+
+  call-atlas:
+    uses: ./.github/workflows/realm.yml
+    secrets: inherit
+    with:
+      sha: ${{ github.sha }}
+      environment: staging
```

```suggestion
      environment: production
```

I understand we need the environment on GitHub allocated first, but want to catch it here too.